discovery-provider/.env

```diff
@@ -8,6 +8,6 @@ audius_ipfs_port=6001
 audius_ipfs_gateway_hosts=https://cloudflare-ipfs.com,https://ipfs.io
 audius_discprov_block_processing_window=100
 audius_discprov_peer_refresh_interval=10
-audius_discprov_identity_service_url=http://localhost:8000
+audius_discprov_identity_service_url=http://docker.for.mac.localhost:7000
```

Comment:
now im so confused about how this was functioning before...thanks for the find

Reply:
dunno! neither here nor there